index.js

```diff
@@ -6,6 +6,13 @@ module.exports = {
   included: function(app) {
     this._super.included(app);
 
-    app.import('vendor/style.css');
+    var appImport;
```

Comment:
And we'll ride off into the sunset of ES6

![image](https://cloud.githubusercontent.com/assets/4205/17520967/2d8cadae-5e17-11e6-8bed-d7038e9aedcb.png)